GVRf/Extensions/gvrf-particlesystem/src/main/res/raw/particle_vert.vsh

```diff
@@ -35,14 +35,7 @@ void main() {
             snoise(posn.xyz + vec3(43.0, 17.0, deltaTime)),
         	snoise(posn.xyz + vec3(-17.0, -43.0, deltaTime))), 0);
 
-    #ifdef HAS_MULTIVIEW
-           bool render_mask = (u_render_mask & (gl_ViewID_OVR + uint(1))) > uint(0) ? true : false;
-           mat4 mvp = u_mvp_[gl_ViewID_OVR];
-           gl_Position = mvp  * posn;
-    #else
-       	    gl_Position =  u_mvp * posn;
-    #endif
-
+    gl_Position =  u_mvp * posn;
```

Comment:
The HAS_MULTIVIEW ifdef block is really not needed anymore?

Reply:
No, this is no longer necessary because the Java shader generation code takes care of it now.